cmd/environment/environment.go.old

```diff
@@ -0,0 +1,182 @@
+// package environment
```

Comment:
This looks like it should be removed

Reply:
I removed this in the most recent commit!